swagger.yml

```diff
@@ -125,6 +234,29 @@ definitions:
             type: string
           reviews:
             type: string
+  Review:
+    type: object
+    required:
+      - message
+      - user
+    properties:
+      id:
+        description: "Unique book id"
+        type: string
+      last_updated:
+        description: "UTC timestamp of when the review was last updated"
+        type: string
```

Comment:
For `last_updated` we can add more information such as `format: date-time` and  an`example: "2020-04-26T08:05:52Z"`